src/components/LoginForm.tsx

```diff
@@ -0,0 +1,37 @@
+'use client'
+import { addAuthUser } from '@/store/actions/auth-action';
+import { RootState } from '@/store/store';
+import * as React from 'react'
+import { useSelector } from 'react-redux';
+import { useDispatch } from 'react-redux';
+import { Dispatch } from 'redux';
+
+const LoginForm: React.FC = () => {
+  const [userId, setUserId] = React.useState('')
+  
+  const authState = useSelector((state: RootState) => state.reducerAuth.auth)
+
+  const dispatch: Dispatch<any> = useDispatch()
```

Comment:
try to not use "any" type.

Reply:
Done. I needed to decide which type to use instead because our code examples also use <any>. But I read about it, and then I replaced by the actions type: AuthAction for login, UserAction for addUser, for example.